ha/k8s_setup/ha_setup.py

```diff
@@ -221,7 +225,7 @@ def process(self):
             num_pods = Conf.get(self._index, f'cortx{_DELIM}common{_DELIM}product_release')
             num_pods = '10' #temporary value till the same is avilabe in cluster.conf
 
-            conf_file_dict = {'LOG' : {'path' : const.HA_LOG_DIR, 'level' : const.HA_LOG_LEVEL},
+            conf_file_dict = {'LOG' : {'path' : ha_log_path, 'level' : const.HA_LOG_LEVEL},
```

Comment:
The right approach is to change RA_LOG_DIR (ha/const.py) and HA_LOG_DIR (in /ha/k8s_setup/const.py) to point to the path mentioned in cluster.conf.

The SUPPORT BUNDLE GENERATION etc pickup the files form RA_LOG_DIR folder directly so these logs will not be available there

Reply:
As discussed created tkt to replace **RA_LOG_DIR (ha/const.py) and HA_LOG_DIR (in /ha/k8s_setup/const.py)**
https://jts.seagate.com/browse/EOS-27352

Reply:
What was the log path in cluster.conf for the unittest output mentioned above? Please mention it and this comment can be resolved - closed.

Reply:
Log path mentioned in cluster.conf - **/share/var/log/cortx/**

Reply:
Thanks for confirmation,